python_embed.c

```diff
@@ -1499,7 +1500,11 @@ void replace_item_core(gentity_t* ent, int item_id) {
 static PyObject* PyMinqlx_ReplaceItems(PyObject* self, PyObject* args) {
     PyObject *arg1, *arg2 ;
     int entity_id = 0, item_id = 0;
+    #if PY_MINOR_VERSION < 7
```

Comment:
True statement for Python 3.5, 3.6, 4.0, 4.1 .. 4.6.
Suggesting to use PY_VERSION_HEX